dbaas-catalog.yaml

```diff
@@ -7,6 +7,8 @@ spec:
   displayName: DBaaS Platform Catalog
   publisher: DBaaS Community
   sourceType: grpc
+  grpcPodConfig:
```

Comment:
We need to remove it atm because it will break everything for dbaas

Reply:
not until you merge it to the `percona-platform` branch

Reply:
and this file is fine https://github.com/percona/dbaas-catalog/blob/percona-platform/percona-dbaas-catalog.yaml you should worry about and it has that fix, maybe we should restore that in main:
https://github.com/percona/dbaas-catalog/blob/main/percona-dbaas-catalog.yaml

`dbaas-catalog.yaml` supposed to be upstream version and shouldn't be used in DBaaS